### PR TITLE
Edited container storage types section

### PR DESCRIPTION
**Pull request by moisesjgomez**
Edited container storage types section to use bullet points for better visibility

### CODE REVIEW

**Comment by cmcclister** [comment]
Learn Build status updates of commit _[e940393](https://github.com/moisesjgomez/cloud-adoption-framework/commits/e9403933a38a5e5376adc45b8ee9a78a3c896ba8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md](https://github.com/moisesjgomez/cloud-adoption-framework/blob/patch-2/docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/0286ad65-6ec8-0c70-0cae-2edc1504c701/202211301721006600-1013/BuildReport?accessString=ba333587774097849cd7fe46b4d76fab130e81e87e2db92344cb9c559cafbbb9).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0286ad65-6ec8-0c70-0cae-2edc1504c701?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by Jak-MS** [comment]
@BrianBlanchard 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.

Note that the numbering should be fixed to conform to markdown syntax ... (eg. `1.` not `1`) 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by igorpag — APPROVED** [review]

**Comment by igorpag** [comment]
#sign-off

File: docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md [hunk 1]
```diff
@@ -125,7 +125,10 @@ We recommend that you incorporate these best practices into your infrastructure
 For most scenarios, we recommend that you use Azure Files or Azure NetApp Files to store FSLogix user profile and Office containers.
 
 - Split user profile and Office containers.
-- We recommend these options for container storage types, in this order: Azure Files Premium tier, Azure NetApp Files Standard tier, and Azure NetApp Files Premium tier.
+- We recommend these options for container storage types, in this order: 
+  1) Azure Files Premium tier
+  2) Azure NetApp Files Standard tier
+  3) Azure NetApp Files Premium tier
```
Comment (Jak-MS) [inline docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md:131]:
```suggestion
- We recommend these options for container storage types, in this order: 

   1. Azure Files Premium tier
   1. Azure NetApp Files Standard tier
   1. Azure NetApp Files Premium tier
```

**Comment by cmcclister** [comment]
Learn Build status updates of commit _[541e54f](https://github.com/moisesjgomez/cloud-adoption-framework/commits/541e54f2c36fcd7f09249285d1c2fe456dc61aaf)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-735449db)

<a id="735449db"></a>
###
 - **[Error: GitBranchDeletedOrForcePushed]** 
`````
Cannot sync git repo to specified commit because ref +refs/pull/1013/merge has been deleted or has been force pushed.
 If this is pull request, please ensure it is mergeable (no merge conflict) and open (not closed).
`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/0286ad65-6ec8-0c70-0cae-2edc1504c701/202212011527097574-1013/BuildReport?accessString=74eb936af7d155ef9f5179bd2be02b7a376fc91fbe2463b9310e3c7f36ecab82).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0286ad65-6ec8-0c70-0cae-2edc1504c701?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Pull request merged**